skopt/callbacks.py

```diff
@@ -285,6 +286,13 @@ class CheckpointSaver(object):
     >>> skopt.gp_minimize(obj_fun, [(-2, 2)], n_calls=10,
     ...                   callback=[checkpoint_callback]) # doctest: +SKIP
 
+
+    When re-using stored results.
+    >>> checkpoint_callback = skopt.callbacks.CheckpointSaver("./result.pkl")
+    >>> x0,y0 = checkpoint_callback.load_previous_evaluations()
```

Comment:
As indicated in https://github.com/scikit-optimize/scikit-optimize/pull/998, we also need to warm start `base_estimator=` to continue where we had left off.